### PR TITLE
Goow 5

### LINKED ISSUE #36: Create the trigger for calling combat
Need to figure out when during overworld travel we want to trigger a combat sequence

### PR DESCRIPTION
**Pull request by beserkzombie**

### CODE REVIEW

File: Global.gd [hunk 1]
```diff
@@ -67,7 +67,7 @@ func goto_scene(target_scene):
 	# we can be sure that no code from the current scene is running:
 	match target_scene:
 		Scene.OVERWORLD:
-			call_deferred("_deferred_goto_scene", Scene.OVERWORLD, "res://Overworld.tscn")
+			call_deferred("_deferred_goto_scene", Scene.OVERWORLD, "res://overworld/Overworld.tscn")
```
Comment (demwilson) [inline Global.gd:70]: Glad to see overworld moved to its own directory.

File: overworld/Player.gd [hunk 2]
```diff
@@ -14,13 +14,24 @@ var Raycasts = {
...
-var previousDir
+
+var stepsTaken = 0
+var stepsToTriggerCombat = 20
+var encounterStep = 20
+
 signal collided
 func _ready():
-	overworld = get_parent()
+	overworld = get_node("../..")
```
Comment (demwilson) [inline overworld/Player.gd]: I think this is the only relative path I've seen you create. From a consistency point of view, does it need to be done this way, or can you do this with an absolute path?

**Review by demwilson — CHANGES_REQUESTED** [review]

File: Global.gd (hunk 1 above)
Comment (beserkzombie) [reply Global.gd:70]: fixed

File: overworld/Player.gd (hunk 2 above)
Comment (beserkzombie) [reply overworld/Player.gd]: fixed

**Review by demwilson — APPROVED** [review]

**Pull request merged**